yeat/bandage/bandage.py

```diff
@@ -0,0 +1,38 @@
+# -------------------------------------------------------------------------------------------------
+# Copyright (c) 2022, DHS. This file is part of YEAT: http://github.com/bioforensics/yeat
+#
+# This software was prepared for the Department of Homeland Security (DHS) by the Battelle National
+# Biodefense Institute, LLC (BNBI) as part of contract HSHQDC-15-C-00064 to manage and operate the
+# National Biodefense Analysis and Countermeasures Center (NBACC), a Federally Funded Research and
+# Development Center.
+# -------------------------------------------------------------------------------------------------
+
+from pkg_resources import resource_filename
+from snakemake import snakemake
+import subprocess
+import warnings
+
+
+def check_bandage():
+    completed_process = subprocess.run(["Bandage", "--help"], capture_output=True, text=True)
+    if completed_process.returncode == 1:
+        print(completed_process.stderr)
+        return False
+    return True
```

Comment:
When Bandage is not found in the users' $PATH, I have it print out the error message caught by subprocess.run().

I also when had this function return a boolean value based on the return code.

If the command was successful, the return code is 0--not 1. If failed, the return code is 1. (Makes sense.)

For the sake of readability when the function returns whether the command worked, instead of returning the return code, I returned back true or false accordingly.